lms/static/js/views/fields.js

```diff
@@ -116,8 +141,8 @@
             },
 
             showSuccessMessage: function () {
-                var successMessage = this.getMessage('success');
-                this.showNotificationMessage(successMessage);
+                var successMessage = this.getMessage('success').toString();
```

Comment:
Ah, I see. They should definitely be standardized to be the same thing I think. Is there actual html content in the return value of `getMessage()`? If so, HtmlSnippets would be best, but if there's no html to be seen, strings would be easier.